### ISSUE TITLE
"Reset all preferences" button border appearing in green color in High contrast black mode and brown color in High contrast white mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#345]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode before launching the application.

1. Launch Xamarin Workbooks application. Which invokes "Create New Workbook" window.
2. Navigate to Tools -> Preferences. Invoke preferences tab. It will open the general tab of preferences.
3. Observe the border color of "Reset all preferences" button.

### Expected Behavior
Border color should follow high contrast standards
### Actual Behavior
"Reset all Preferences" button border is appearing in green color in High contrast balck and brown color in high contrast white, but it is not a disabled button. User may perceive it as disabled button because of green color border.
### User Impact
As the border color appears in green color in HC Black and brown in HC White, user may perceive it as disabled element which is actually not. It deviates the user from the purpose.

### Screenshots
![8 hc black](https://user-images.githubusercontent.com/22956130/40723067-9e170944-643b-11e8-97e2-2ffb17c552c5.PNG)
![8 hc white](https://user-images.githubusercontent.com/22956130/40723069-9e42455a-643b-11e8-8b44-ff261f54b164.PNG)
![8 normal mode](https://user-images.githubusercontent.com/22956130/40723070-9e6ef1b8-643b-11e8-940e-91bce82a2389.PNG)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
